### PR TITLE
Improve upon existing code as per review comments

### PR DESCRIPTION
**Pull request by ayush000**
- Replace `for` with `doseq`
- Replace `if` block with `when` block
- Don't return anything in main function
- Read list of items to watch from a JSON file

### CODE REVIEW

File: src/github_amazon_bot/core.clj [hunk 1]
```diff
@@ -26,13 +24,12 @@
...
-           (api/send-text token chat-id (str name " is available! Please buy it from " url))
-           (catch Exception ex true)) true)))))
+    (doseq [ipad ipads :let [name (ipad :name)
+                             url (ipad :url)
+                             body (get-site-body url)
+                             ipad-is-available (not (is-text-present? text-to-search body))]]
+      (when ipad-is-available
+        (try
+          (api/send-text token chat-id (str name " is available! Please buy it from " url))
+          (catch Exception))))))
```
Comment (rubaljabbal) [inline src/github_amazon_bot/core.clj]: You need catch Exceptions in a var.

File: src/github_amazon_bot/core.clj [hunk 2]
```diff
@@ -16,20 +17,25 @@
...
+                             url (item :url)
                              body (get-site-body url)
-                             ipad-is-available (not (is-text-present? text-to-search body))]]
-      (when ipad-is-available
+                             item-is-available (not (is-text-present? text-to-search body))]]
+      (when item-is-available
         (try
           (api/send-text token chat-id (str name " is available! Please buy it from " url))
-          (catch Exception))))))
+          (catch Exception ex))))))
```
Comment (rubaljabbal) [inline src/github_amazon_bot/core.clj:39]: I see that you've fixed it already :)

**Review by rubaljabbal — APPROVED** [review]
LGTM

File: src/github_amazon_bot/core.clj (hunk 2 above)
Comment (ayush000) [reply src/github_amazon_bot/core.clj:39]: Linter was showing a warning for unused var. Not sure how to fix that, so leaving it as is.

**Pull request merged**